Project-Aurora/Project-Aurora/Devices/Hue/HueDevice.cs

```diff
@@ -92,7 +92,10 @@ public bool Initialize()
                                 client.Initialize(key);
                                 File.WriteAllText(Path.Combine(Global.AppDataDirectory, bridge.BridgeId), key);
                             }
-                            catch { }
+                            catch
+                            {
+                                //Just catch the error RegisterAsync throws if the sync button isn't pressed yet
```

Comment:
Its an empty because RegisterAsync will throw an error if the sync button is not pressed witch would spam the log.
In essentially it would bloat the log file with unnecessary lines and would probably bloat it to over megabytes or even gigabytes depending on when you press the sync button.

Reply:
as seen here https://github.com/Q42/Q42.HueApi/blob/master/README.md#bridge in the comment above RegisterAsync